src/Expensify.js

```diff
@@ -222,4 +237,5 @@ export default compose(
             key: ONYXKEYS.SCREEN_SHARE_REQUEST,
         },
     }),
+    withNetwork(),
```

Comment:
This will re-render entire app whenever network status changes.
To prevent re-render, I thin it's better to subscribe onyx network in libs/actions/Report.js
This is fine because `isOffline` value is checked from deeplink only on initial app load and doesn't require listening real time network status change.

Reply:
Oh thanks for catching that @situchan! I will check it out shortly

Reply:
Updated!